statsmodels/tsa/statespace/tests/test_representation.py

```diff
@@ -601,34 +602,47 @@ def empty_endog():
 def test_bind():
     # Test binding endogenous data to Kalman filter
 
-    mod = Representation(1, k_states=2)
+    mod = Representation(2, k_states=2)
 
     # Test invalid endogenous array (it must be ndarray)
     assert_raises(ValueError, lambda: mod.bind([1,2,3,4]))
 
     # Test valid (nobs x 1) endogenous array
-    mod.bind(np.arange(10)*1.)
-    assert_equal(mod.nobs, 10)
+    mod.bind(np.arange(10).reshape((5,2))*1.)
+    assert_equal(mod.nobs, 5)
```

Comment:
I did that following a suggestion above. The issue is that with the new numpy, 1D array are both F and C contiguous ... so the following checks always fail. I changed the main tests to a 2D array, and then added back the 1D array tests if numpy version is less than 1.10.